recOrder/plugin/workers/acquisition_workers.py

```diff
@@ -906,6 +901,54 @@ def work(self):
         self.phase_image_emitter.emit(phase)
         self.meta_emitter.emit(meta)
 
+    def _check_exposure(self) -> None:
+        """
+        Check that all LF channels have the same exposure settings. If not, use the State0 exposure.
+
+        Parameters
+        ----------
+        settings:       (json) JSON dictionary conforming to MM SequenceSettings
+        """
+        logging.debug('Verifying exposure times...')
+        # parse exposure times
+        channel_exposures = []
+        for _, channel in enumerate(self.settings['channels']):
+            channel_exposures.append(channel['exposure'])
+
+        channel_exposures = np.array(channel_exposures)
+        if not np.all(channel_exposures == channel_exposures[0]):
+            # warn user that not all channels exposures were the same
+            logging.warning('The exposure times of each State are not equal!\nAcquiring with the exposure of State0...')
+
+            # setting all channel exposures to the exposure of State0
+            for i in range(len(self.settings['channels']) - 1):
+                self.settings['channels'][i + 1]['exposure'] = self.settings['channels'][0]['exposure']
+        
+        self._check_abort()
+
+    def _acquire(self) -> np.ndarray:
```

Comment:
I'm unsure about pulling this out as it's own function since it's only 3 lines, it only gets called once, and it those three lines do more than just "acquire". I'm also unsure about this refactor since it doesn't match the other classes in this file.

Reply:
It gets called two in two places (2D and 3D) so I think reducing repetitive code might save some future errors.

Reply:
Roger that...makes sense. Thanks!